src/ripple/app/ledger/Ledger.h

```diff
@@ -420,6 +416,13 @@ class Ledger
         return mHash;
     }
 
+    // Updates the fees cached in the ledger.
+    // Safe to call concurrently. We shouldn't be storing
+    // fees in the Ledger object, they should be a local side-structure
+    // associated with a particular module (rpc, tx processing, consensus)
+    //
+    void deprecatedUpdateCachedFees() const;
```

Comment:
I'm missing the larger picture then. It does not need to be public and a function that updates an internal cache would normally be private. Where are you going with this that it needs to be public?